lib/matplotlib/font_manager.py

```diff
@@ -50,6 +50,26 @@
 
 _log = logging.getLogger(__name__)
 
+
+class findfontFilter(logging.Filter):
```

Comment:
I would consider making this a private class, as this is internal functionality added to an already private logger.

(This is what `stubtest` is complaining about, as it sees this as public but not added to the stub files)